quote/syn update is broken for macros

```rust
   Compiling cssparser-macros v0.3.1 (file:///home/brain/tmp/rust-cssparser/macros)
warning: unused import: `quote::ToTokens`
  --> macros/lib.rs:13:5
   |
13 | use quote::ToTokens;
   |     ^^^^^^^^^^^^^^^
   |
   = note: #[warn(unused_imports)] on by default

warning: unused import: `TokenNode`
  --> macros/lib.rs:14:19
   |
14 | use proc_macro2::{TokenNode, TokenStream, TokenTree};
   |                   ^^^^^^^^^

error[E0277]: the trait bound `syn::Expr: std::fmt::Debug` is not satisfied
  --> macros/lib.rs:28:66
   |
28 |             _ => panic!("expected a match expression, got {:?}", expr)
   |                                                                  ^^^^ `syn::Expr` cannot be formatted using `:?`; if it is defined in your crate, add `#[derive(Debug)]` or manually implement it
   |
   = help: the trait `std::fmt::Debug` is not implemented for `syn::Expr`
   = note: required by `std::fmt::Debug::fmt`

error[E0277]: the trait bound `syn::Pat: std::fmt::Debug` is not satisfied
  --> macros/lib.rs:34:78
   |
34 |                 _ => panic!("expected string or wildcard pattern, got {:?}", pattern)
   |                                                                              ^^^^^^^ `syn::Pat` cannot be formatted using `:?`; if it is defined in your crate, add `#[derive(Debug)]` or manually implement it
   |
   = help: the trait `std::fmt::Debug` is not implemented for `syn::Pat`
   = note: required because of the requirements on the impl of `std::fmt::Debug` for `&syn::Pat`
   = note: required by `std::fmt::Debug::fmt`

error[E0277]: the trait bound `syn::PatLit: std::fmt::Debug` is not satisfied
  --> macros/lib.rs:42:66
   |
42 |                 _ => panic!("expected string pattern, got {:?}", expr)
   |                                                                  ^^^^ `syn::PatLit` cannot be formatted using `:?`; if it is defined in your crate, add `#[derive(Debug)]` or manually implement it
   |
   = help: the trait `std::fmt::Debug` is not implemented for `syn::PatLit`
   = note: required because of the requirements on the impl of `std::fmt::Debug` for `&syn::PatLit`
   = note: required by `std::fmt::Debug::fmt`

error: aborting due to 3 previous errors

error: Could not compile `cssparser-macros`.
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -12,6 +12,7 @@ script:
   - cargo test --features dummy_match_byte
   - if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then cargo test --features bench; fi
   - if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then cargo test --features "bench dummy_match_byte"; fi
+  - cd macros && cargo build --verbose
 
 notifications:
   webhooks: http://build.servo.org:54856/travis

--- a/macros/Cargo.toml
+++ b/macros/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "cssparser-macros"
-version = "0.3.1"
+version = "0.3.2"
 authors = ["Simon Sapin <simon.sapin@exyr.org>"]
 description = "Procedural macros for cssparser"
 documentation = "https://docs.rs/cssparser-macros/"
@@ -15,5 +15,5 @@ proc-macro = true
 procedural-masquerade = {path = "../procedural-masquerade", version = "0.1"}
 phf_codegen = "0.7"
 quote = "0.4"
-syn = {version = "0.12", features = ["full"]}
+syn = {version = "0.12", features = ["full", "extra-traits"]}
 proc-macro2 = "0.2"

